UKHomeOffice/sgmr-service
# SPCR-145 Rename “Submit a pleasure craft report” to “Tell Border Force and HMRC you are sailing to or from the UK in a pleasure craft” (4)

## Description
We have new user flows being tested with users. In the meantime, we should take the content on the prototype and update the existing pages on the service to match.

Renamed "Submit a pleasure craft report” to “Tell Border Force and HMRC you are sailing to or from the UK in a pleasure craft” throughout the service

## To Test
For each page in the website
- Ctrl + F and search for the phrase "Submit a pleasure craft report" in your browser
- You should get no results
## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
